profile/2017-03-09-profile.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
+        },
+        "microsoft.keyvault": {
+            "version": "2016-10-01",
+            "types": {}
+        },
+        "microsoft.network": {
+            "version": "2015-06-15",
+            "types": {}
```

Comment:
How do we handle the Not supported resource type in an api-version? e.g though azurestack supports 2015-06-15 api version for microsoft.network,  it does not support applicationGateway resource type

Reply:
It depends how you interpret this file IMHO. Is this a Stack only file or not?
If not, and we're saying "Everything that is under Microsoft.Network should be called with 2015-06-15" and that's fine for Public Azure. Stack gateway will complain that the RT doesn't exist, and it's a fine error as well.

Reply:
If someone builds an app on this profile it should work with both Stack and Azure - that's the promise of profiles. Profiles themselves are generic, Stack happens to snap to a specific profile.

An app developer might build an app with a profile on public Azure, then expect it to run on Stack. For that case I think we need to somehow spec that a resource type isn't supported in a given profile; that's only part of the problem though, the SDKs aren't designed to exclude use of certain types I don't think.

Reply:
This is an Open issue 2